### PR TITLE
Disable the ability to apply for inactive job seekers

### PR DESCRIPTION
**Pull request by kemar**
- Add a custom template for users who attempt to login while their account is inactive
- Disable the ability to apply for inactive job seekers

### CODE REVIEW

File: itou/www/apply/forms.py [hunk 1]
```diff
@@ -18,11 +18,17 @@ def __init__(self, *args, **kwargs):
...
-            error = _(
-                "Vous ne pouvez pas postuler pour cet utilisateur car il n'est pas demandeur d'emploi."
-            )
-            raise forms.ValidationError(error)
+        if self.user:
+            if not self.user.is_active:
+                error = _(
+                    "Vous ne pouvez pas postuler pour cet utilisateur car son compte a été désactivé."
+                )
+                raise forms.ValidationError(error)
```
Comment (vgrange) [inline itou/www/apply/forms.py]: 👌

File: itou/templates/account/account_inactive.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{# django-allauth template override. #}
+{% extends "layout/content_small.html" %}
```
Comment (vgrange) [inline itou/templates/account/account_inactive.html]: pour ma curiosité comment marche ce tour de magie? Tu overwrite magiquement juste en te basant sur le filename `account_inactive`?

https://github.com/pennersr/django-allauth/blob/master/allauth/templates/account/account_inactive.html

Comment (kemar) [reply itou/templates/account/account_inactive.html]: C'est le mécanisme du backend de templates de Django qui permet ça : https://docs.djangoproject.com/en/dev/howto/overriding-templates/#overriding-from-the-project-s-templates-directory

C'est documenté ici dans `allauth` :

- https://django-allauth.readthedocs.io/en/latest/templates.html#overridable-templates
- https://django-allauth.readthedocs.io/en/latest/views.html#login-account-login

Comment (vgrange) [reply itou/templates/account/account_inactive.html]: 👍

**Pull request merged**